appveyor.yml

```diff
@@ -15,18 +15,30 @@ environment:
     - nodejs_version: "11.15"
     - nodejs_version: "12.22"
     - nodejs_version: "13.14"
-    - nodejs_version: "14.18"
+    - nodejs_version: "14.20"
+    - nodejs_version: "15.14"
+    - nodejs_version: "16.20"
+    - nodejs_version: "17.9"
+    - nodejs_version: "18.19"
+    - nodejs_version: "19.9"
+    - nodejs_version: "20.11"
+    - nodejs_version: "21.6"
+    - nodejs_version: "22.0"
 cache:
   - node_modules
 install:
   # Install Node.js
   - ps: >-
       try { Install-Product node $env:nodejs_version -ErrorAction Stop }
-      catch { Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) }
+      catch { Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) x64 }
   # Configure npm
   - ps: |
       npm config set loglevel error
-      npm config set shrinkwrap false
+      if ((npm config get package-lock) -eq "true") {
```

Comment:
**Issue**: The conditional check for 'package-lock' configuration is incorrect. The 'npm config get package-lock' command returns a string, not a boolean. This can lead to incorrect configuration settings being applied. <br> **Fix**: Correct the conditional check to compare against the string 'true' instead of a boolean. <br> **Code Suggestion**: 
 ```
 - if ((npm config get package-lock) -eq "true") {
 + if ((npm config get package-lock) -eq "true") {
 ```